cmd/dep/signals.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
I'm not very certain about this file. `HandleSignals` and `Release` are methods on *SourceMgr that aren't defined in the SourceManager interface, but which are used by commands. They seem pretty concrete, so I figured these interface assertions might be an okay way forward. I'm interested in feedback.